app/vmctl/vm_native.go

```diff
@@ -7,13 +7,16 @@ import (
 	"io"
 	"log"
 	"net/http"
+	"sync"
 	"time"
 
-	"github.com/cheggaaa/pb/v3"
-
+	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmctl/backoff"
+	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmctl/barpool"
 	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmctl/limiter"
 	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmctl/stepper"
 	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmctl/vm"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+	"github.com/cheggaaa/pb/v3"
```

Comment:
vmnative became too big already. I think, it should be moved to a separate package and used in a fashion like the rest of exporters are used.

Reply:
I moved parts of the code to their package. Simplified vm_native.go